packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -277,38 +277,6 @@ String? _readFileContent(File file) {
   return file.existsSync() ? file.readAsStringSync() : null;
 }
 
-const String _androidPluginRegistryTemplateOldEmbedding = '''
-package io.flutter.plugins;
-
-import io.flutter.plugin.common.PluginRegistry;
-{{#methodChannelPlugins}}
-import {{package}}.{{class}};
-{{/methodChannelPlugins}}
-
-/**
- * Generated file. Do not edit.
- */
-public final class GeneratedPluginRegistrant {
-  public static void registerWith(PluginRegistry registry) {
-    if (alreadyRegisteredWith(registry)) {
-      return;
-    }
-{{#methodChannelPlugins}}
-    {{class}}.registerWith(registry.registrarFor("{{package}}.{{class}}"));
```

Comment:
Interesting that this is where the lint in https://discord.com/channels/608014603317936148/846507907876257822/1214337088657293333 was coming from